about.html

```diff
@@ -0,0 +1,104 @@
+      </div>
+    </nav>
+    <nav class="desktop-menu">
+      <a href="index.html"><img src="images/logo-en.svg" alt="logo culture days" width="100" height="64"></a>
+      <ul class="main-menu">
+        <li class="menu-list"><a href="about.html">About</a></li>
+        <li class="menu-list"><a href="index.html#program">Program</a></li>
+        <li class="menu-list"><a href="index.html#join">Join</a></li>
+        <li class="menu-list"><a href="index.html#sponsor">Sponsor</a></li>
+        <li class="menu-list"><a href="index.html#news">News</a></li>
+        <li class="campaign-menu"><a href="index.html">Campaign</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main class="cultural-days-wrap">
+    <section class="about">
+      <h1 class="about-tittle">ABOUT CULTURE DAYS</h1>
+      <div class="bar-icon"></div>
+      <div class="about-wrap">
+        <p class="invite-msg about-msg">Culture Days is a national celebration of arts and culture. At the end of each September, millions of people attend thousands of free participatory arts and culture events across the country both in-person and online. Culture Days programs invite the public to get hands-on and behind-the-scenes to highlight the importance of arts and culture in our communities.</p>
+      </div>
+      <p class="invite-msg contact-msg">Please contact us per Email for any questions about Culture Days 2022!</p>
+      <a class="about-email" href="#">info@culturedays.ca</a>
+    </section>
+    <section class="featured-activities logo">
+      <h2 class="secondary-tittle featured-tittle logo-tittle">Creative Culture Days Logo</h2>
+      <div class="bar-icon bar-icon-about"></div>
+      <p class="invite-msg about-msg logo-msg">The Culture Days logo is the foundation of our visual identity. It is bold and impactful. The pops of bright colour contrast the black letterforms and help bring the logo to life. The elements have been specially designed and must not be altered in any way.</p>
+      <div class="logo-wrap">
+        <img src="images/logo-en.svg" alt="logo culture days" width="200" height="200">
+      </div>
+      <div class="bg-color-icon-1"></div>
+      <div class="bg-color-icon-2"></div>
+      <div class="bg-color-icon-3"></div>
+      <div class="bg-color-icon-4"></div>
+      <div class="bg-color-icon-5"></div>
+    </section>
+    <section class="featured-activities logo">
+        <h2 class="secondary-tittle featured-tittle logo-tittle">See the past Culture Days</h2>
+        <div class="bar-icon bar-icon-about"></div>
+        <p class="invite-msg about-msg logo-msg">Take a look at the Culture Days</p>
+        <div class="img-about-wrap">
+          <img class="past-img" src="https://cdn.whistler.com/s3/url/july-long-weekend/showcase/first-nations-canada.jpg" width="539" height="298">
+          <img class="past-img" src="https://s24748.pcdn.co/wp-content/uploads/2020/10/Acadian-dancers.jpg" width="539" height="298">
+        </div>
+      </section>
```

Comment:
- [ ] I think it would be better if you add a section in your **abiut.html** with a design like in the image below to make your page identical as much as possible to the expected design.

![Screenshot from 2022-09-14 20-11-01](https://user-images.githubusercontent.com/83042742/190230583-3ef5bb23-f620-4859-9970-cc9e641c23ff.png)

- [ ] Also make sure you add the text on each card and use at least the correct **fonts(size && weight)** to match to the **[Behance template design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)**. Check everything on the design and try to stick to it as much as possible